### ISSUE TITLE
Game crashes while changing the volume of a Speaker Pole

### ISSUE THREAD

**Issue by Bommik** [CoderDE/FicsIt-Networks#58]
There is a bug with the Speaker Pole. If you change the volume while the sound is playing, the game crashes.  
I tried it in release and development build 37 both have this bug.  
This code should reproduce the bug.
```
speaker = component.proxy("foo")  
speaker:playSound("bar")  
event.pull(1)  
speaker:setVolume(1)

**Comment by Panakotta00** [comment]
Can plz try to cause the crash and the send  a debug file you can generate with SMM.

**Comment by Bommik** [comment]
[SMMDebug_2020-8-7_1-31-24.zip](https://github.com/CoderDE/FicsIt-Networks/files/5183034/SMMDebug_2020-8-7_1-31-24.zip)

**Comment by Panakotta00** [comment]
Should now be fixed with version 0.0.8
